dev/com.ibm.ws.cdi.api_fat/test-applications/injectInjectionPointAsParam.war/src/com/ibm/ws/fat/cdi/injectInjectionPointParam/InjectInjectionPointAsParamServlet.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.fat.cdi.injectInjectionPointParam;
+
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+
+@WebServlet("/")
+public class InjectInjectionPointAsParamServlet extends FATServlet {
+
+    private static final long serialVersionUID = 1L;
+
+    @Inject
+    TestBean bean;
+
+    @Test
+    @Mode(TestMode.FULL)
+    public void testInjectInjectionPointAsParam() throws IOException {
+        if (!bean.test()) {
+            fail("Bean Manager and/or Injection Point not correctly injected into TestBean constructor");
+        }
```

Comment:
Probably better to do asserts inside `TestBean.test()` so you have a better idea of what actually failed.